pillar/environment-prod-public.sls

```diff
@@ -13,3 +13,8 @@ elife_xpub:
     s3:
         bucket: prod-elife-xpub
     deployment_target: prod
+
+journal:
+    feature_xpub: false
+    submit_url: http://submit.elifesciences.org/
```

Comment:
Better as `https`.

Reply:
I copied this from https://elifesciences.org, but it supports HTTPS. Was it intentional?

Reply:
https://github.com/elifesciences/builder-configuration/blob/master/pillar/journal-public.sls#L2 is indeed https, but has never been used.

Reply:
Journal looks never to have been updated to make it HTTP (suspect it wasn't available over HTTPS at the time). Was going to make the switch now.